spec/models/maneuver_participant_spec.rb

```diff
@@ -1,4 +1,32 @@
 require 'rails_helper'
-RSpec.describe ManeuverParticipant, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+describe ManeuverParticipant do
+  describe 'score' do
+    it'score should be 22' do
```

Comment:
What's the significance of the number 22? It's difficult to tell from reading this test.